src/components/atoms/Info/Info.tsx

```diff
@@ -70,6 +70,7 @@ const Info: FC<IInfoProps> = ({ infoText, disabled, size = "smallNudge", appeara
         <Tooltip text={infoText} alwaysShow={alwaysShow} appearance={appearance === "inverse" ? "inverse" : "default"}>
             <button
                 type="button"
+                aria-label="press enter to open tooltip"
```

Comment:
The aria-label="press enter to open tooltip" could be improved for better accessibility and clarity. Here’s why:

1. Not all users rely on Enter: While Enter is commonly used for activation, users may also use Spacebar or other assistive technology interactions.
2. "Press enter" is instructional: ARIA labels should describe the element, not instruct interaction methods.
3. It’s not clear what "tooltip" refers to: A more descriptive label would help users understand the purpose of the button.


```suggestion
               aria-label="More information"
```